bundle/Resources/public/js/views/cof-contentcreationview.js

```diff
@@ -409,6 +417,28 @@ YUI.add('cof-contentcreationview', function (Y) {
              * @type Object
              */
             selectedLocation: {},
+
+            /**
+             * Should prevent from closing discovery widget?
+             *
+             * @attribute preventClosingDiscoveryWidget
+             * @type Boolean
+             * @default true
+             */
+            preventClosingDiscoveryWidget: {
```

Comment:
This attribute name sounds like a method. Maybe, this name: `closingDiscoveryWidgetPrevented` sounds better?